contracts/modules/LoanOpenings.sol

```diff
@@ -639,26 +647,27 @@ contract LoanOpenings is
      *     receiver: receiver of funds (address(0) assumes borrower address).
      *     manager: delegated manager of loan unless address(0).
      * @param sentValues The values to send:
-     *     newRate: New loan interest rate.
+     *     interestRate: New loan interest rate.
      *     newPrincipal: New loan size (borrowAmount + any borrowed interest).
-     *     torqueInterest: New amount of interest to escrow for Torque loan (determines initial loan length).
+     *     interestInitialAmount: New amount of interest to escrow for Torque loan (determines initial loan length).
      *     loanTokenReceived: Total loanToken deposit (amount not sent to borrower in the case of Torque loans).
-     *     collateralTokenReceived: Total collateralToken deposit.
+     *     collateralTokenSent: Total collateralToken deposit.
+     *     minEntryPrice: Minimum entry price for checking price divergence (Value of loan token in collateral).
      * @return The loanId.
      * */
     function _initializeLoan(
         LoanParams memory loanParamsLocal,
         bytes32 loanId,
         uint256 initialMargin,
-        address[4] memory sentAddresses,
-        uint256[5] memory sentValues
+        MarginTradeStructHelpers.SentAddresses memory sentAddresses,
+        MarginTradeStructHelpers.SentAmounts memory sentValues
```

Comment:
we only need `newPrincipal` from  `MarginTradeStructHelpers.SentAmounts` - can be replaced with only one value

Reply:
Good idea